save-frontend/src/main/resources/locales/cn/proposing.json

```diff
@@ -7,7 +7,6 @@
   "Original vulnerability identifier": "原始漏洞ID",
   "Related link": "相关链接",
   "Language": "语言",
-  "Criticality": "危险程度",
```

Comment:
why removed?

Reply:
This field has been removed from the frontend

Reply:
In this PR or in another one? Because I don't see these changes here